frontend/ui/data/css_tweaks.lua

```diff
@@ -24,8 +24,9 @@ local T = require("ffi/util").template
 -- Default globally enabled style tweaks, for new installations
 local DEFAULT_GLOBAL_STYLE_TWEAKS = {}
 -- Display in-page per-specs footnotes for EPUB and FB2:
-DEFAULT_GLOBAL_STYLE_TWEAKS["footnote-inpage_epub_smaller"] = true
+DEFAULT_GLOBAL_STYLE_TWEAKS["footnote-inpage_epub"] = true
 DEFAULT_GLOBAL_STYLE_TWEAKS["footnote-inpage_fb2"] = true
+DEFAULT_GLOBAL_STYLE_TWEAKS["inpage_footnote_font-size_smaller"] = true
```

Comment:
Another thing that could be done is switching this default to `inpage_footnote_font-size_80`.

That would apply to new installs, whereas existing users get the "legacy" tweak `inpage_footnote_font-size_smaller` via the migration.

Maybe a good idea if it's strongly recommended to use the newer tweaks as @poire-z suggests. Then we'd also be more likely to get reports if these newer tweaks are worse in some situation. After some time one could migrate everyone from the legacy tweak (if wanted).

Reply:
As you feel.